### PR TITLE
Beta versions npm publisher

### PR DESCRIPTION
**Pull request by temujin**
* increment beta version on every patch to 'develop' branch
    except version bamping (compares package.json versions from
    current and previous patches).
    NOTE: this solution uses latest published version (catched from npm)
          and increments it. No commits, tags will be pushed back to github.
  * publish beta package to npm

    NOTE: !!! This solution is for CI/CD only. You'll get amended commit and
           bumped version of package.json in case of local use !!!
  * in this patch implemented CD of latest development version.
    Thus docker based build toolkit removed.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -15,4 +15,18 @@ before_script:
...
\ No newline at end of file
+  - '[ "${TRAVIS_PULL_REQUEST}" = "false" ] && sh deploy.sh || true'
+
+after_success:
+  .cicd/beta_npm_ver.js
+
+deploy:
+  skip_cleanup: true
+  provider: npm
+  email:
```
Comment (valorkin) [inline .travis.yml]: @temujin @angieskazka @jheeffer 
email and api_key has to be set before merging

**Comment by valorkin** [comment]
@angieskazka @jheeffer please review and merge
it will became a deploy blocker

**Comment by korel-san** [comment]
:+1:

**Comment by rychkog** [comment]
:+1:

**Pull request merged**